airflow/jobs/scheduler_job_runner.py

```diff
@@ -422,7 +423,15 @@ def _executable_task_instances_to_queued(self, max_tis: int, session: Session) -
             task_instance_str = "\n".join(f"\t{x!r}" for x in task_instances_to_examine)
             self.log.info("%s tasks up for execution:\n%s", len(task_instances_to_examine), task_instance_str)
 
+            tis_we_have_room_for = self._slots_free_for_tis(task_instances_to_examine)
```

Comment:
LOL.   Naming things is hard.  :heart:

Reply:
I actually quite like these explicit variable names :)